### PR TITLE
Adding option to install Anaconda3

### PR DESCRIPTION
**Pull request by nareynolds**
Hi,

I just wanted to add the ability to install the version of Anaconda that runs Python 3.4 by default. In the process, I also need to include .DS_store in the .gitignore.

Cheers,
Nathaniel

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,6 @@ bin/*
 
 # anaconda installs should not be checked in
 Anaconda-*-Linux-*.sh
+
+# OSX
+.DS_Store
```
Comment (thmttch) [inline .gitignore]: Oops, forgot this. Thanks!

**Comment by thmttch** [comment]
This looks good to me, let me just run the tests (which take forever), and then I'll merge.

Thanks for the contribution!

**Comment by thmttch** [comment]
Awesome, works for me as well.

**Comment by thmttch** [comment]
I'll cut a release for this as well.

**Comment by nareynolds** [comment]
Glad I could contribute. Thanks so much for this awesome cookbook!

**Pull request merged**